.changeset/strong-otters-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+"single-spa-react": patch
+---
+
+fix: types and exports of the package
```

Comment:
maybe update this to be more clear that this will allow typescript inference to work in more situations

Reply:
Fixed.